### PR TITLE
Update templates.yml for bumblebee

### PR DESCRIPTION
**Pull request by jaikev**

### CODE REVIEW

**Comment by jaikev** [comment]
Angel please check this

**Comment by malonnnn** [comment]
bumblebee is ambiguous, please be more specific on which hero this is

**Comment by jaikev** [comment]
Hes like a helper of Optimus Prime

**Comment by jaikev** [comment]
oh nvm.

File: Citizens/templates.yml [hunk 1]
```diff
@@ -1,3 +1,35 @@
+bumblebee(dc):
```
Comment (malonnnn) [inline Citizens/templates.yml]: paranthesis not allowed here

**Pull request closed without merging**